themes/cupper/static/css/styles.css

```diff
@@ -799,192 +1228,265 @@ h1 svg {
+        min-height: auto;
     }
 
     #patterns-list {
-      margin-top: 0.5rem;
-      border: 1px solid;
+        margin-top: 0.5rem;
+        border: 1px solid;
     }
 
     .patterns h3 {
         font-size: 1.5rem;
         padding: 1.5rem 1rem 0.75rem;
     }
+
     .patterns li:not(.pattern) {
         margin-top: 0;
     }
+
     .patterns ul ul {
         margin: 0;
     }
+
     .patterns li {
         margin-top: 0;
     }
+
     .pattern {
         font-size: 1rem;
     }
+
     .pattern a {
         padding: 1rem;
     }
+
     .pattern [aria-current] {
         clip-path: none;
         padding: 1rem;
     }
-    .pattern + .pattern {
+
+    .pattern+.pattern {
         border-top: 1px solid;
         margin-top: 0;
     }
+
     #menu-button {
         display: block;
     }
+
     #patterns-list {
         display: none;
     }
+
     .toc-link {
-      display: none;
+        display: none;
     }
-    [aria-expanded="true"] + #patterns-list {
+
+    [aria-expanded="true"]+#patterns-list {
         display: block;
     }
+
     code {
-      word-break: break-all;
+        word-break: break-all;
     }
 }
+
 @media print {
+
     .wrapper:not(.print-version) .intro-and-nav,
     .wrapper:not(.print-version) [role="contentinfo"] {
         display: none;
     }
+
     .main-and-footer {
         margin-left: 0;
     }
+
     a {
         border: 0;
     }
+
     main a::after {
         content: " (" attr(href) ")";
         word-break: break-word;
     }
+
     main nav a::after {
         content: '';
     }
+
     .cmd code {
         background: #fefefe;
         color: #111;
     }
+
     pre code {
         white-space: pre-wrap !important;
     }
+
     .expandable-section {
         border: 0;
         padding: 0;
     }
-    .expandable-section + p {
+
+    .expandable-section+p {
         margin-top: 0.75rem;
     }
+
     [id^="js-expandable-"] {
         display: block;
     }
+
     [data-expands] svg {
         display: none;
     }
+
     main *:not(.with-desc) {
         page-break-inside: avoid;
     }
+
     .note.warning {
         border-left: 0.5rem solid;
         background: none;
     }
 }
+
 @media (-ms-high-contrast: active) {
-  img[src*=".svg"] {
-    background: #fefefe;
-    padding: 0.5rem;
-  }
-  .ticks li::before {
-    content: '✓';
-    background: none;
-    width: auto;
-    top: 0;
-  }
-  .note.warning {
-      border-left: 0.5rem solid;
-      background: none;
-  }
+    img[src*=".svg"] {
+        background: #fefefe;
+        padding: 0.5rem;
+    }
+
+    .ticks li::before {
+        content: '✓';
+        background: none;
+        width: auto;
+        top: 0;
+    }
+
+    .note.warning {
+        border-left: 0.5rem solid;
+        background: none;
+    }
 }
+
+/* Customize */
+
+.main-and-footer {
+    background: url("/images/line-art-science-researcher.svg");
+    background-repeat: no-repeat;
+    background-size: 66%;
+    background-position: left center;
```

Comment:
I think this styles can be improved to match the design better. Firstly, not adding the logo as a background image will help. Also, probably adding some offset in background-position.

Reply:
ooo yeah not having the logo as a background image and a regular image would be great.